mlir/lib/Conversion/VectorToAMDGPU/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+add_mlir_conversion_library(MLIRVectorToAMDGPU
+  VectorToAMDGPU.cpp
+
+  ADDITIONAL_HEADER_DIRS
+  ${MLIR_MAIN_INCLUDE_DIR}/mlir/Conversion/VectorToAMDGPU
+
+  DEPENDS
+  MLIRConversionPassIncGen
+
+  LINK_COMPONENTS
+  Core
+
+  LINK_LIBS PUBLIC
+  MLIRAMDGPUDialect
+  MLIRVectorDialect
+  MLIRPass
+  MLIRTransforms
+  )
```

Comment:
Can you also register this in bazel? We will need it downstream in IREE.

Reply:
Do you mean to ask to register this pass downstream once this PR is merged pulled into iree? For sure. Since I'm going to pull this pass into IREE immediately after this happens, I kinda have to do this to make IREE CI happy.

Reply:
Ahh I was completely unaware of this until you reminded. Will do!

Reply:
This is added and tested locally.

It took me a while to realize the base commit of this branch is bad and keep erroring out on vector transform tablegen target. I have to cherry-pick the base commit to tip of llvm-project to be able to build and verify my change.